index.html

```diff
@@ -0,0 +1,1942 @@
+						  Trump's
+						  former national security adviser during the impeachment inquiry. </div>
+					  </article>
+					  <article class="more-story-article">
+						<div class="image">
+						  <a href="#"><img class="img-fluid" src="images/more-story6.jpg"
+							  alt="Trump Iran descalation"></a>
+						</div>
+						<h4><a href="#"></a>Canadian CEO Blames 'Narcissist' Trump for Plane Shot Down</h4>
+						<div class="article-summary">
+						  Michael McCain said the 176 people killed aboard Ukraine International Airlines Flight 752 were
+						  "collateral damage" of President Donald Trump's "irresponsible" behavior. </div>
+					  </article>
+					  <article class="more-story-article">
+						<div class="image">
+						  <a href="#"><img class="img-fluid" src="images/more-story7.jpg"
+							  alt="Trump Iran descalation"></a>
+						</div>
+						<h4><a href="#"></a>Prince Harry and Prince William Jointly Deny U.K. Newspaper Bullying Rumor</h4>
+						<div class="article-summary">
+						  A spokesperson for the princes said a story about the relationship between the brothers is
+						  "false."
+						</div>
+					  </article>
+					  <article class="more-story-article">
+						<div class="image">
+						  <a href="#"><img class="img-fluid" src="images/more-story8.jpg"
+							  alt="Trump Iran descalation"></a>
+						</div>
+						<h4><a href="#"></a>'No Way' Esper Wouldn't Have Seen Seen Same Iran Intel as Trump: McFaul</h4>
+						<div class="article-summary">
+						  Mark Esper appeared to contradict the president on Sunday, when he said he had not seen evidence
+						  that
+						  Iran
+						  was planning attacks on four American embassies. </div>
+					  </article>
+					  <article class="more-story-article">
+						<div class="image">
+						  <a href="#"><img class="img-fluid" src="images/more-story9.jpg"
+							  alt="Trump Iran descalation"></a>
+						</div>
+						<h4><a href="#"></a>HIV May Be Getting More Infectious as a Result of 'Natural Selection'</h4>
+						<div class="article-summary">
+						  "Viruses, like all organisms, adapt and evolve," co-author Joel Wertheim told Newsweek. </div>
+					  </article>
+					  <article class="more-story-article">
+						<div class="image">
+						  <a href="#"><img class="img-fluid" src="images/more-story10.jpg"
+							  alt="Trump Iran descalation"></a>
+						</div>
+						<h4><a href="#"></a>Oscar Nominations 2020: Fans Express Outrage as Lupita Nyong'o Snubbed</h4>
+						<div class="article-summary">
+						  The 2020 Oscar nominations have been released, and Twitter called out some of the Academy's
+						  oversights
+						  of
+						  women and people of color. </div>
+					  </article>
+	
+					</div>
+				  </div>
+				  <div class="culture-story-content order-2">
+					<div class="content">
+					  <div class="head-story">
+						<div class="title">
+						  <a href="#">Culture & Travel</a>
+						</div>
+					  </div>
```

Comment:
- [ ] This section should have three articles in the larger and medium screen sizes. Yours have two. Please adjust them.